packages/makesheet/sheetInfo.js

```diff
@@ -0,0 +1,20 @@
+function doGet(e) {
+  const sheetId = e.parameter.sheetId;
+  if (!sheetId) {
+    return createJsonResponse({ error: "sheetId parameter required" });
+  }
+  
+  try {
+    const file = DriveApp.getFileById(sheetId);
```

Comment:
`DriveApp.getFileById` is ambient authority. Note the convention for that:

```js
function doGet(e, io = {}) {
    const {
      getFileById = DriveApp.getFileById,
    } = io;
    ...
    getFileById(sheetId)
 ```

This lets the caller inject an implementation for testing, or interpose an attenuated version, etc.

Please add a CONTRIBUTING.md with a section on this.
excerpt from https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline as appropriate